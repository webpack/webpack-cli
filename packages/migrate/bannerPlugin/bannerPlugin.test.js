"use strict";

const defineTest = require("@webpack-cli/utils//defineTest");

defineTest(__dirname, "bannerPlugin", "bannerPlugin-0");
defineTest(__dirname, "bannerPlugin", "bannerPlugin-1");
defineTest(__dirname, "bannerPlugin", "bannerPlugin-2");
