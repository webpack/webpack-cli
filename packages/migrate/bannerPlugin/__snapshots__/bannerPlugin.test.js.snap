// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bannerPlugin transforms correctly using "bannerPlugin-0" data 1`] = `
"module.exports = {
    plugins: [
        new webpack.BannerPlugin({
            raw: true,
            entryOnly: true,
            banner: 'Banner'
        })
    ]
}
"
`;

exports[`bannerPlugin transforms correctly using "bannerPlugin-1" data 1`] = `
"// Should do nothing if there is no banner plugin
module.exports = {
    plugins: []
}
"
`;

exports[`bannerPlugin transforms correctly using "bannerPlugin-2" data 1`] = `
"// Only transform if it uses the old format
module.exports = {
    plugins: [
        new webpack.BannerPlugin({})
    ]
}
"
`;
